client/post-editor/editor-sidebar/header.jsx

```diff
@@ -10,26 +10,49 @@ import Gridicon from 'gridicons';
  */
 import Button from 'components/button';
 import EditorPostType from 'post-editor/editor-post-type';
+import {
+	NESTED_SIDEBAR_NONE,
+	NESTED_SIDEBAR_REVISIONS,
+	NestedSidebarPropType,
+} from './util';
 
-function EditorSidebarHeader( { translate, toggleSidebar } ) {
-	return (
-		<div className="editor-sidebar__header">
+const EditorSidebarHeader = ( { nestedSidebar = NESTED_SIDEBAR_NONE, toggleSidebar, translate } ) => (
+	<div className="editor-sidebar__header">
+		{ nestedSidebar === NESTED_SIDEBAR_REVISIONS && (
+			<span>
+				<Button
+					borderless
+					className="editor-sidebar__header-title"
+					onClick={ toggleSidebar }
+					title={ translate( 'Close sidebar' ) }
+				>
+					<EditorPostType isSettings />
+				</Button>
+				<span>
+					→ { translate( 'Revisions' ) }
```

Comment:
by "here", you mean in untranslated JSX? Otherwise, there are a couple of (good) precedents:

```javascript
analytics.pageView.record(
	paths.domainManagementPrimaryDomain( ':site', ':domain' ),
	'Domain Management › Set Primary Domain'
);
```
[client/my-sites/upgrades/domain-management/controller.js](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/upgrades/domain-management/controller.jsx#L67)

```javascript
<span className="process-orders-widget__order-label">
	{ translate( '✨ New orders' ) }
</span>
```
[client/extensions/woocommerce/components/process-orders-widget/index.js
](https://github.com/Automattic/wp-calypso/blob/master/client/extensions/woocommerce/components/process-orders-widget/index.js#L23)

And quite a lot of "ellipsis" in various places.